qa/tasks/ceph_manager.py

```diff
@@ -301,12 +301,14 @@ def kill_osd(self, osd=None, mark_down=False, mark_out=False):
             # import
             cmd = (prefix + "--op import --file {file}")
             cmd = cmd.format(id=imp_osd, file=exp_path)
-            proc = imp_remote.run(args=cmd, wait=True, check_status=False)
-            output = proc.stderr.getvalue()
-            bogosity = "The OSD you are using is older than the exported PG"
-            if proc.exitstatus == 1 and bogosity in output:
-                self.log("OSD older than exported PG"
-                         "...ignored")
+            proc = imp_remote.run(args=cmd, wait=True, check_status=False,
+                                  stderr=StringIO())
+            stderr_content = proc.stderr.getvalue()
```

Comment:
no need to define a `stderr_content`, it is not referenced otherwhere besides line 309.